.gitlab-ci.yml

```diff
@@ -246,6 +256,48 @@ dbnd_airflow 0 1_10_9_py36:
       junit: modules/dbnd-airflow/build/junit-py36-airflow_1_10_9.xml
   only: *on_significant_branch
 
+dbnd_tensorflow 1 py27:
+  <<: *template__test_tensorflow
+  variables:
+    IMAGE: python:2.7
+    TEST_NAME: py27-tensorflow1
+
+dbnd_tensorflow 1 py35:
+  <<: *template__test_tensorflow
+  variables:
+    IMAGE: python:3.5
+    TEST_NAME: py35-tensorflow1
+
+dbnd_tensorflow 2 py35:
+  <<: *template__test_tensorflow
+  variables:
+    IMAGE: python:3.5
+    TEST_NAME: py35-tensorflow2
+
+dbnd_tensorflow 1 py36:
+  <<: *template__test_tensorflow
+  variables:
+    IMAGE: python:3.6
+    TEST_NAME: py36-tensorflow1
+
+dbnd_tensorflow 2 py36:
+  <<: *template__test_tensorflow
+  variables:
+    IMAGE: python:3.6
+    TEST_NAME: py36-tensorflow2
+
+dbnd_tensorflow 1 py37:
```

Comment:
let's remove it from the ci/cd and keep only tensorflow 1 + py27/py36,   tensorflow 2 + py36

(all others are still available in tox)

Reply:
okay. whats the reason for that?

Reply:
minimize Ci/CD jobs we need to run. ( right now I think it will cover 99% of the problems)

Reply:
we can just mark them as manual steps for now

Reply:
good idea, done